nautobot/dcim/api/serializers.py

```diff
@@ -199,6 +199,19 @@ class Meta:
         list_display_fields = ["name", "status", "parent", "tenant", "description", "tags"]
         # https://www.django-rest-framework.org/api-guide/validators/#optional-fields
         validators = []
+        # TODO: Add documentation of this schema
+        detail_view_config = [
+            {
+                "Location": {"fields": ["name", "rack_count", "parent"]},
+                "Management": {"fields": ["status", "prefix_count", "device_count"]},
+                "Others": {"fields": ["tenant"]},
+            },
+            {
+                "Tags": {"fields": ["tags"]},
```

Comment:
As we move toward more and more models having `tenant` and `tags` fields, it'd be good to have these handled in a standardized way too, I think?